Formula/xray.rb

```diff
@@ -41,6 +46,44 @@ def install
     resource("geosite").stage do
       pkgshare.install "dlc.dat" => "geosite.dat"
     end
+
+    resource("example_config").stage do
+      pkgetc.install "vless_tcp_xtls.json" => "example_config.json"
+    end
+  end
+
+  def caveats
+    <<~EOS
+      An example_config.json is installed in #{etc}/xray.
+      For more, visit https://github.com/XTLS/Xray-examples.
+      `brew services` requires a config file at #{etc}/xray/config.json
+    EOS
+  end
+
+  plist_options manual: "xray run --config #{HOMEBREW_PREFIX}/etc/xray/config.json"
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>KeepAlive</key>
+          <true/>
+          <key>RunAtLoad</key>
+          <true/>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{opt_bin}/xray</string>
+            <string>run</string>
+            <string>--config</string>
+            <string>#{etc}/xray/config.json</string>
```

Comment:
Does it not run without specifying this config?

Reply:
It will otherwise try to read config from stdin

Reply:
So `brew services start xray` will wait for input from stdin? That's not good, and will definitely leave many users of this formula quite confused. A default config that will work out of the box is needed here, I think.

Reply:
By specifying plist this way, a fresh installation will cause `brew services start xray` to error, saying that `${etc}/xray/config.json` doesn’t exist.